Ghostscript.NET.Viewer/app.config

```diff
@@ -1,3 +1,12 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Can we remove the changes in this fie?

Reply:
Done. I just chose the wrong commit description in [1e83b70](https://github.com/ArtifexSoftware/Ghostscript.NET/pull/128/commits/1e83b7074346d74029a8c0091e8816e24aecba94)